charts/cloudzero-agent/templates/cm.yaml

```diff
@@ -159,7 +159,11 @@ data:
       {{- end}}
   {{- end}}
     remote_write:
+      {{ if .Values.aggregator.enabled }}
+      - url: 'http://{{ include "cloudzero-agent.aggregator.name" . }}.{{ .Release.Namespace }}.svc.cluster.local/metrics'
```

Comment:
so, `/metrics` is the endpoint that the prometheus agent is remote writing to on the collector? do you think it would make sense to change it a bit? just because `/metrics` is used for prometheus scraping usually, and we will probably want to scrape metrics from the collector at some point. I know this is a POST vs a GET, but might be nice just for clarity.